.gitmodules

```
@@ -1,3 +1,4 @@
 [submodule "docs/tools/gitwash"]
 	path = docs/tools/gitwash
 	url = https://github.com/matthew-brett/gitwash.git
+	branch = master
```

```suggestion
	branch = main
```
My bad, the branch is called `main`, not `master` in `gitwash` repo.